README.md

```diff
@@ -0,0 +1,252 @@
+PAYU expects this submodel's configuration files in a subdirectory with that name.
+
+    collate:
+       exe: /short/public/access-esm/payu/bin/mppnccombine
+       restart: true
+       mem: 4GB
+
+Ask Aidan
+
+    restart: /short/public/access-esm/payu/restart/pre-industrial
+
+Ask Aidan
+
+    calendar:
+        start:
+            year: 101
+            month: 1
+            days: 1
+
+        runtime:
+            years: 1
+            months: 0
+            days: 0
+
+Here is the start date, and the runtime **per run**.
+The total time you want to model is `runtime` * `number of runs`
+
+    runspersub: 5
+
+This `runspersub` feature is a nifty tool to allow you to bundle several runs into a single submission for the PBS queue.
+
+Let's have an example: Say you told payu to make 7 runs with the above setting.
+Each run would have a runtime of 1 year. So in the first submission it would run the model 5 times, to model years 101 through 105 respectively.
+
+Then it would automatically resubmit another pbs job to model years 106 and 107, and then end.
+
+### Setting up the Atmosphere Submodel
+
+The **name** in `config.yaml` for the atmosphere submodel is "atmosphere", so the configuration of the UM will be in the `atmosphere` subdirectory.
+
+    $ ls atmosphere/
+    CNTLALL   SIZES      __pycache__  ftxx       ihist          prefix.CNTLATM
+    CONTCNTL  STASHC     cable.nml    ftxx.new   input_atm.nml  prefix.CNTLGEN
+    INITHIS   UAFILES_A  errflag      ftxx.vars  namelists      prefix.PRESM_A
+    PPCNTL    UAFLDS_A   exstat       hnlist     parexe         um_env.py
+
+There are many configuration files, but I want to note the `um_env.py`.
+This file is used to set environment variables for the UM.
+The UM driver of PAYU will look for this file and execute it.
+
+### Setting up the Ocean Submodel
+
+The **name** in `config.yaml` for the ocean submodel is "ocean", so the configuration
+of MOM will be in the `ocean` subdirectory.
+
+    $ ls ocean
+    data_table  diag_table  field_table  input.nml
+
+
+### Setting up the Ocean Submodel
+
+The **name** in `config.yaml` for the ice submodel is "ice", so the configuration
+of CICE will be in the `ice` subdirectory.
+
+    $ ls ice/
+    cice_in.nml  input_ice.nml
+
+## Running the Model
+
+If you have set up the modules system to use the `/g/data3/hh5/public/modules` folder, a simple `module load payu` should give you access to the payu system.
+
+From the control directory, type
+
+    $ payu setup
+
+This will prepare a the model run based on the configuration of the experiment.
+It will setup `work` and `archive` directories and link to them from within the
+configuration directory.
+You don't have to do that, as the run command also sets it up, but it helps to check for errors.
+
+    $ payu sweep
+
+This command removes the `work` directory again, but leaves the `archive`.
+You can have it remove the `archive` as well by appending `--hard` to the command.
```

Comment:
Not sure this is a good thing to mention. A hard sweep will remove all the users archived data, with NO undo option.